fastify.js

```diff
@@ -207,8 +210,6 @@ function fastify (options) {
     decorateRequest: decorator.decorateRequest,
     hasRequestDecorator: decorator.existRequest,
     hasReplyDecorator: decorator.existReply,
-    // middleware support
-    use: use,
```

Comment:
I merged next, this line should remain since the linting is breaking cause of this

Reply:
The code is now breaking because in the last merge we lost
```js
  // We are adding `use` to the fastify prototype so the user
  // can still access it (and get the expected error), but `decorate`
  // will not detect it, and allow the user to override it.
  Object.setPrototypeOf(fastify, { use })
```
I'll add it back.